python/cudf/cudf/testing/dataset_generator.py

```diff
@@ -237,13 +238,8 @@ def get_dataframe(parameters, use_threads):
 
     # For each column, use a generic Mimesis producer to create an Iterable
```

Comment:
Update this comment since mimesis is not being used.

Reply:
Nice catch. Removed